.github/workflows/build_SRWA.yml

```diff
@@ -0,0 +1,89 @@
+# This workflow Builds the UFS SRWEATHER APP
+name: Build SRWA
+
+on:
+  push:
+    branches: [ feature/CI_build_test ]
+  pull_request:
+    branches: [ rrfs_ci, feature/CI_build_test ]
+
+# Use a default login shell which loads intel/oneapi/setvars.sh, and lmod path 
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+ 
+  Clone_rrfs_ci_pcluster:
+    # Auxillary repository needed to insert env files to run CI tests in the short term 
+    name: Clone RRFS CI repo
+    # This job will run on AWS Parallel Cluster head node
+    runs-on: self-hosted
+    steps:
+    
+    - name: Checkout robgonzalezpita/rrfs-ci-pcluster Repo
+      uses: actions/checkout@v3
+      with:
+        repository: robgonzalezpita/rrfs-ci-pcluster
+        path: ./rrfs-ci-pcluster
+  
+  Configure_SRWA:
+    name: Checkout SRWA in parent directory 
+    runs-on: self-hosted 
+    needs: [ Clone_rrfs_ci_pcluster ]
+
+    steps:
+      # Checkout the SRWA - in parent directory, with commit sha-1 in filepath to avoid multiple PR Conflicts, & track builds by commit
+      # Specify branch used for testing, if there is not a similar named branch in ufs-SRWA
+      - name: Checkout SRWA 
+        run: |
+          cd ../
+          mkdir ${{ github.sha }} && cd ${{ github.sha }} 
+          git clone https://github.com/NOAA-GSL/ufs-srweather-app.git
+          cd ufs-srweather-app
+          exists=`git show-ref ${{ github.ref }}`
+          if [ -n "$exists" ]; then
+            echo 'branch exists!'
+            git checkout ${{ github.ref_name }}
+          else
+            git checkout rrfs_ci
```

Comment:
Is there a way to keep from hard-coding this so deep in the workflow? Maybe even just pulling it out to a more prominent place will help us remember to change it to the most appropriate default branch name?

Reply:
Potentially, I'll look into this a little bit more. I'm not quite sure where a more prominent place would be. Are you thinking more prominent just to remember to change rrfs_ci back to develop? Or are you thinking we'll want to have a variable for the branch we're submitting the PR against?

Reply:
Yep, I changed `rrfs_ci` to ${{ github.base_ref }}